packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -19,6 +19,7 @@
 	margin-bottom: $grid-unit-20;
 	background: $gray-100;
 	border-radius: $radius-block-ui;
+	overflow: hidden;
```

Comment:
This stops the `Aa` text breaking out of the font preview container when the font size is really large.